package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "umami-analytics-next",
+  "version": "1.0.0",
+  "description": "A simple type-safe integration between the Umami Analytics API, loaded using Next's Script component",
+  "keywords": [
+    "analytics",
+    "next",
+    "react",
+    "typescript",
+    "umami"
+  ],
+  "bugs": "https://github.com/hemilabs/umami-analytics-next/issues",
+  "license": "MIT",
+  "author": "Gonzalo D'Elia <gonzalo@bloq.com>",
+  "files": [
+    "_esm/**/*",
+    "_types/**/*",
+    "src/**/*.ts",
+    "src/**/*.tsx"
+  ],
+  "repository": "hemilabs/umami-analytics-next",
+  "scripts": {
```

Comment:
Most of the benefit is on Typescript land (DX experience, and compilation safety), so no tests. For a javascript user, it is just a context that loads the Script and a hook that returns `window.umami`, there is nothing much to test (on javascript).

I tried to find a way to test the typings (using [tsd](https://www.npmjs.com/package/tsd)), but as the typings are defined on the parameters `track` accepts, instead of the response, I couldn't figure how to properly test it.